roles/installer/templates/storage/metrics.yaml.j2

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ deployment_type }}-metrics-claim
+  namespace: {{ ansible_operator_meta.namespace }}
+  ownerReferences: null
+  labels:
+    {{ lookup("template", "../common/templates/labels/common.yaml.j2") | indent(width=4) | trim }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ metrics_storage_size | default('5Gi') }}
```

Comment:
PVC's are often hard to resize unless you have a storage class that supports dynamic resizing.  Maybe we should set this a bit higher?  Depends on how much data we expect here.